packages/fxa-payments-server/server/lib/routes/legal-docs.js

```diff
@@ -0,0 +1,184 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+const config = require('../../config');
+const joi = require('joi');
+const fetch = require('node-fetch');
+const Sentry = require('@sentry/node');
+const { cdnFqdn, resultsCacheLimit } = config.get('legalDocLinks');
+const acceptLanuageParser = require('accept-language-parser');
+const supportedLanguages = require('fxa-shared/l10n/supportedLanguages.json');
+const DEFAULT_LOCALE = 'en';
```

Comment:
The GitHub issue says the default locale should be `'en-US'`. Does it make a difference here?

Reply:
The difference is that the legal docs repo doesn't have an en-US.  I changed this from 'en-US' after noticing that in the legal docs repo.  See https://github.com/mozilla/legal-docs/